backend/services/store.service.js

```
@@ -1,20 +1,95 @@
+                });
+                let _close_time = set(now, {
+                    hours: close_time.getHours(),
+                    minutes: close_time.getMinutes(),
+                    seconds: 0,
+                    milliseconds: 0,
+                });
+                // 深夜営業を考慮
+                // 例：16:00〜05:00
+                if (_open_time >= _close_time) {
+                    if (now < _close_time) _open_time = addDays(_open_time, -1);
+                    _close_time = addDays(_close_time, 1);
```

Biome (1.9.4)

[error] Can't assign _open_time because it's a constant

This is where the variable is defined as constant

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)